eventmesh-common/src/test/java/org/apache/eventmesh/common/file/WatchFileManagerTest.java

```diff
@@ -50,11 +50,18 @@ public boolean support(FileChangeContext changeContext) {
         };
         WatchFileManager.registerFileChangeListener(f.getParent(), fileChangeListener);
 
-        Properties properties = new Properties();
-        properties.load(new BufferedReader(new FileReader(file)));
-        properties.setProperty("eventMesh.server.newAdd", "newAdd");
-        FileWriter fw = new FileWriter(file);
-        properties.store(fw, "newAdd");
+        try (FileReader fr = new FileReader(file);
+            BufferedReader br = new BufferedReader(fr);
+            FileWriter fw = new FileWriter(file)) {
```

Comment:
Good job.

When using `.close()` to close nested streams, we only need to close the outermost stream, because the close request propagates from the outermost stream to the innermost stream.

To have try-with-resources close them, you assigned the streams to variables as you open them, instead of nesting. If you use nesting, an exception during construction of one of the later streams (e.g. `GZIPOutputStream`) will leave any stream constructed by the nested calls inside it open.

Reply:
Thanks for the review :)